spec/std/process_spec.cr

```diff
@@ -321,4 +321,42 @@ describe Process do
       Process.find_executable("some_very_unlikely_file_to_exist").should be_nil
     end
   end
+
+  describe "quote_posix" do
+    it { Process.quote_posix("").should eq "''" }
+    it { Process.quote_posix(" ").should eq "' '" }
+    it { Process.quote_posix("$hi").should eq "'$hi'" }
+    it { Process.quote_posix(orig = "aZ5+,-./:=@_").should eq orig }
+    it { Process.quote_posix(orig = "cafe").should eq orig }
+    it { Process.quote_posix("café").should eq "'café'" }
+    it { Process.quote_posix("I'll").should eq %('I'"'"'ll') }
+    it { Process.quote_posix("'").should eq %(''"'"'') }
+    it { Process.quote_posix("\\").should eq "'\\'" }
+
+    context "join" do
+      it { Process.quote_posix([] of String).should eq "" }
+      it { Process.quote_posix(["my file.txt", "another.txt"]).should eq "'my file.txt' another.txt" }
+      it { Process.quote_posix(["foo ", "", " ", " bar"]).should eq "'foo ' '' ' ' ' bar'" }
+      it { Process.quote_posix(["foo'", "\"bar"]).should eq %('foo'"'"'' '"bar') }
+    end
+  end
+
+  describe "quote_windows" do
+    it { Process.quote_windows("").should eq %("") }
+    it { Process.quote_windows(" ").should eq %(" ") }
+    it { Process.quote_windows(orig = "%hi%").should eq orig }
```

Comment:
Is this really correct? Wouldn't `%hi%` expand an environment variable?

Reply:
```
  # NOTE: This is **not** safe to pass to the CMD shell.
```
https://github.com/crystal-lang/crystal/pull/9043#issuecomment-612642548

Reply:
Please add this to the method documentation!

Reply:
Hm, maybe I just didn't understand it correctly what's already there. So you can leave it as is unless someone else finds this should be more detailled.